src/containers/StructurePage/folderComponents/topicMenuOptions/SetResourcesPrimary.tsx

```diff
@@ -47,7 +52,7 @@ const SetResourcesPrimary = ({ node, editModeHandler: { editMode, toggleEditMode
     <>
       <MenuItemButton stripped onClick={toggleConnectedResourcesPrimary}>
         <RoundIcon small icon={<Pencil />} />
-        {t('taxonomy.resourcesPrimary.buttonText')}
+        {t('taxonomy.resourcesPrimary.buttonText', { count: recursive ? 2 : 1 })}
```

Comment:
Hadde dette vært mer lesbart dersom vi hadde to forskjellige translation-navn?

Reply:
Har oppdatert med nye phrases.